src/SDDPoptimize.jl

```diff
@@ -169,14 +164,14 @@ Return:
 Float64 (estimation of the upper bound)
 
 """
-function estimate_upper_bound(model, param, V, problems, n_simulation=1000)
+function estimate_upper_bound(model::SPModel, param::SDDPparameters, V::Vector{PolyhedralFunction}, problem::Vector{JuMP.Model}, n_simulation=1000::Int)
```

Comment:
I would think that problems make more sense than problem as it is a vector of problem... Open to discussion.